pkg/gadgets/tcptracer/tracer/core/bpf/tcptracer.bpf.c

```diff
@@ -243,7 +267,10 @@ int BPF_KPROBE(entry_trace_close, struct sock *sk)
 	if (!fill_tuple(&tuple, sk, family))
 		return 0;
 
-	fill_event(&tuple, &event, pid, uid, family, TCP_EVENT_TYPE_CLOSE);
+	task = (struct task_struct*)bpf_get_current_task();
+	mntns_id = (u64) BPF_CORE_READ(task, nsproxy, mnt_ns, ns.inum);
```

Comment:
`fill_event()` will be called by the same thread which does the TCP related operation, right?
So, what about doing this into `fill_event()`?

Reply:
I modified the code to pass `mntns_id` into `filter_event()`, so I think it's fine to pass this variable to `fill_event()` too.